scripts/build-run-e2e-cov.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
```

Comment:
Where is this script used, is this just for testing?

Reply:
for manual invocation